engine/src/main/java/org/terasology/rendering/opengl/GLSLShader.java

```diff
@@ -124,6 +124,18 @@ int linkShaderProgram(int featureHash) {
         return shaderProgram;
     }
 
+    @Override
+    public void developmentRecompile() {
```

Comment:
This method was renamed in `Shader`, we might need to rename it here to prevent an error (the Jenkins build fails because there is no `developmentRecompile` method in `Shader` to `@Override`)

Reply:
Sorry, my bad. I noticed the mail as I was already off. I'll fix it right away.